src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -754,7 +754,7 @@ private ValueTask PopulateSourceRepoListAsync(IReadOnlyCollection<PackageSourceM
             {
                 SelectedSource = PackageSources
                     // if the old active source still exists. Keep it as the active source.
-                    .FirstOrDefault(i => StringComparer.CurrentCultureIgnoreCase.Equals(i.SourceName, selectedSourceName))
```

Comment:
By time arrive here PackageSources was already filtered to 2 sources, this change actually nop.